### PR TITLE
Check if issued before actually renewing.

### PR DESCRIPTION
**Pull request by d1str0**
### Context

In one or two sentences, what problem is this PR trying to solve?

### Approach

Briefly, how does this PR solve the issue?

### Testing

What was your strategy for testing the new code?

### Misc.

Screenshots? Loose ends? Concerns?

### CODE REVIEW

File: service/acme.go [hunk 1]
```diff
@@ -158,6 +158,9 @@ func (s *acmeService) Trigger(id string) {
 }
 
 func (s *acmeService) Renew(c *model.Certificate) {
+	if !c.Issued {
+		s.Trigger(c.ID)
```
Comment (debus) [inline service/acme.go:162]: Does this go on to renew after starting the trigger? The Trigger is async right? Maybe it should return after calling trigger. We might also want to modify auto renew logic to check if something is issued before renewing

Comment (d1str0) [reply service/acme.go:162]: oh that's weird, it's almost as if code review helps prevent stupid mistakes like this.

**Review by debus — APPROVED** [review]
:+1:

**Comment by d1str0** [comment]
Tested in "prod" and it works. :+1:

**Pull request merged**